build/updatehistorictzdb.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+rm -rf tmp
+mkdir tmp
+
+# Get the list of files, just once.
+
+mkdir -p tmp/releases
+cd tmp/releases
+FILES=files.txt
+
+gsutil.cmd ls gs://nodatime/releases > $FILES
```

Comment:
`gsutil` is just `gsutil` on Linux. Do you need the `.cmd` suffix on Windows?

Either way, perhaps pull out a

```bash
declare -r GSUTIL=gsutil.cmd
```

at the top of the file, then use `${GSUTIL} ls ...` here, that way there's only one place to change.